.github/workflows/build-docker-full.yml

```diff
@@ -30,6 +35,21 @@ jobs:
         with:
           images: salesforce/cli
 
+      - name: Collect Node checksum
+        id: node-checksum
+        uses: salesforcecli/github-workflows/.github/actions/retry@main
+        with:
+          command: |
+            NODE_VERSION="${{ steps.setup-node.outputs.node-version }}"
```

Comment:
I have not tested that this output exists, but it looks like it should: https://github.com/actions/setup-node/blob/main/src/util.ts#L39